lib/rules/web/admin_console/4.4/notification.xyaml

```diff
@@ -0,0 +1,52 @@
+click_notification_drawer:
```

Comment:
This can be used with `click_masthead_button`

Reply:
I would recommend create a new file `masthead.xyaml` with all masthead related rules into it